userspace/libsinsp/plugin.cpp

```diff
@@ -697,6 +697,12 @@ bool sinsp_plugin::resolve_dylib_symbols(void *handle, std::string &errstr)
 			{
 				throw sinsp_exception(string("error in plugin ") + m_name + ": field JSON entry has no name");
 			}
+			const Json::Value &jvdisplay = root[j]["display"];
+			string fdisplay = jvdisplay.asString();
+			if(fdisplay == "")
+			{
+				throw sinsp_exception(string("error in plugin ") + m_name + ": field JSON entry has no display");
+			}
```

Comment:
This is a backward-incompatible API change, so we likely need to bump the [plugin API version](https://github.com/falcosecurity/libs/blob/master/userspace/libscap/plugin_info.h#L31) to 0.4.0

Reply:
Would it be preferable to make the display name optional? I could see where that might be the case for fields with the "hidden" property set.

Reply:
Making it optional might be a good solution here I think!

Reply:
I agree!

Reply:
Done.